public/Day 147/index.html

```
@@ -0,0 +1,1117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="logo.png" type="image/x-icon">
+  <link rel="icon" href="logo.png" type="image/png">
+    <title>VyomDarpan: Bharat’s Own Window to the Cosmos</title>
+    <!-- Load Tailwind CSS -->
+    <script src="https://cdn.tailwindcss.com"></script>
+    <!-- Load KaTeX for high-quality math rendering -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.10/dist/katex.min.css" xintegrity="sha384-wnsB0K5U1I6L0T2Qz9n/E/U7c/7vY7D/n8sWj1F49u3B7wz+hE6/tF0A+9Hl4j" crossorigin="anonymous">
```

HTMLHint (1.9.0)

[warning] The empty tag : [ link ] must be self closed.

(tag-self-close)